rate_limit.py

```diff
@@ -91,12 +91,10 @@ def set_token_server():
 
   set_token_client = """
   function(newToken) {
-    const expiresDateString = new Date(Date.now() + 24 * 60 * 60 * 1000).toUTCString();
-    document.cookie = `arena_token=${newToken}; expires=${expiresDateString};`;
+    localStorage.setItem("arena_token", newToken);
```

Comment:
where is the code getting this token then?

Reply:
token will be stored in [token](https://github.com/yanolja/arena/pull/123/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R83-R84) param.
Each request will send this token [here](https://github.com/yanolja/arena/pull/123/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R192)

Reply:
The token is stored in the memory. Doesn't it mean that you don't have to save it somewhere?

Reply:
Then how could we send the token when a user requests..?